examples/nextjs/components/auth/LoginButton.tsx

```diff
@@ -1,39 +1,21 @@
 import { useWalletLogin, useWalletLogout } from '@lens-protocol/react-web';
 import { useEffect } from 'react';
 import toast from 'react-hot-toast';
-import { useAccount, useConnect, useDisconnect } from 'wagmi';
-import { InjectedConnector } from 'wagmi/connectors/injected';
```

Comment:
Simplifying `next.js` example given it was anyway broken as was using the older `wagmi 0.x`. We anyway have wagmi integration inside `web-wagmi` so to simplify maintenance this is using now just in memory signer as the `react-native` example.

Reply:
nice, thanks!